### PR TITLE
Update Community Cloud trust and security policy

### PR DESCRIPTION
**Pull request by sfc-gh-dmatthews**
## 📚 Context
Update language on Trust and Security page per legal review.

## 🌐 References
[Google doc](https://docs.google.com/document/d/1PpGubWxQ5cIbBVeIFZ61TNTtZ7Z9t1ZBPdTIrfR0dbo/edit)

**Contribution License Agreement**
By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

### CODE REVIEW

**Review by tc87 — APPROVED** [review]
LGTM, thanks for all the helping getting this updated Debbie ♥️

**Review by snehankekre — APPROVED** [review]

**Comment by sfc-gh-dmatthews** [comment]
Thanks @tc87 and @snehankekre. Now I'm just waiting for final confirmation that we're done-done before merging.

**Pull request merged**
